toots/2024-09-02-release-hedgedoc-1-10-0.toot

```diff
@@ -0,0 +1,8 @@
+We've just re-released #HedgeDoc 1.10.0 🎉
+
+⚠️ This release contains a medium severity security fix for users of MySQL/MariaDB!  ⚠️
+
+Please be aware that running this release (or anyother release for that matter) with node 22.7.0 may lead to utf-8 problems and can break your deployment.
+We recommend the latest node 20 for the time being.
```

Comment:
```suggestion
Please be aware that running HedgeDoc with Node.js 22.7.0 may lead to UTF-8 problems because of a regression in Node.
We recommend using Node 20 for the time being.
```